### PR TITLE
Translate to turkish

### PR DESCRIPTION
**Pull request by alibildir**
translated to Turkish language. If browser language is Turkish it changes automatically.

### CODE REVIEW

**Comment by SebastianStehle** [comment]
Awesome. I think there are 2 things that I would like to change:

1. I think it would be good to have a fallback system for the frontend. Because if we add new texts we would not necessarily have everything in turkish.

2. Can also be done later: An option to override the language, independent from the browser setting.

**Comment by alibildir** [comment]
i want to help you. i can do some solutions in a few week. Also if you need add a word or sentences in resx or .ts please add it to .tr. files in english i will check your every commit. maybe you can request code review to me if you need

**Comment by SebastianStehle** [comment]
Resx files already have the fallback handling, but not the frontend. But I can also do that.

File: backend/src/Notifo.Domain/Apps/App.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ namespace Notifo.Domain.Apps;
 
 public sealed record App(string Id, Instant Created)
 {
-    private static readonly ReadonlyList<string> DefaultLanguages = ReadonlyList.Create("en");
+    private static readonly ReadonlyList<string> DefaultLanguages = ReadonlyList.Create("en", "tr");
```
Comment (SebastianStehle) [inline backend/src/Notifo.Domain/Apps/App.cs:19]: This needs to be reverted.

**Review by SebastianStehle — CHANGES_REQUESTED** [review]

**Pull request merged**